cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -2,7 +2,12 @@ include(nuttx/px4_impl_nuttx)
 
 px4_nuttx_configure(HWCLASS m4 CONFIG nsh ROMFS y ROMFSROOT px4fmu_common)
 
-set(CMAKE_TOOLCHAIN_FILE ${PX4_SOURCE_DIR}/cmake/toolchains/Toolchain-arm-none-eabi.cmake)
+
+if("$ENV{PX4_USE_CLANG}" STREQUAL "1")
+	set(CMAKE_TOOLCHAIN_FILE ${PX4_SOURCE_DIR}/cmake/toolchains/Toolchain-arm-none-eabi-clang.cmake)
```

Comment:
It seems like we should do this at another level, perhaps with a cmake option?. Do you only intend to use fmu-v2 initially? Could you do a px4fmu-v2_clang instead and then we'll take another pass to make it generic.